devito/finite_differences/finite_difference.py

```diff
@@ -2,10 +2,11 @@
 
 from functools import partial
 
-from sympy import S, finite_diff_weights
+from sympy import S, finite_diff_weights, Function
```

Comment:
unused?

Reply:
Yes. Was used previously but no longer is.